devops/terraform/bootstrap.sh

```diff
@@ -11,12 +13,11 @@ az storage account create --resource-group "$TF_VAR_mgmt_resource_group_name" \
 --kind StorageV2 --sku Standard_LRS -o table
 
 # Blob container
-SA_KEY=$(az storage account keys list --account-name "$TF_VAR_mgmt_storage_account_name" --resource-group "$TF_VAR_mgmt_resource_group_name" --query "[0].value" -o tsv)
 # shellcheck disable=SC2154
-az storage container create --account-name "$TF_VAR_mgmt_storage_account_name" --name "$TF_VAR_terraform_state_container_name" --account-key "$SA_KEY" -o table
+az storage container create --account-name "$TF_VAR_mgmt_storage_account_name" --name "$TF_VAR_terraform_state_container_name" --auth-mode login -o table
```

Comment:
this was already changed in:
[https://github.com/microsoft/AzureTRE/pull/2607/files](https://github.com/microsoft/AzureTRE/pull/2607/files)